src/viewer.js

```diff
@@ -1027,6 +1033,8 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
 
         // clear our reference to the main element - they will need to pass it in again, creating a new viewer
         this.element = null;
+        this._createdElements = null;
```

Comment:
You don't need to set this to null, since it's no longer being created in the first place.